modules/observability/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * assemblies/assembly-rhdh-observability.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-admin-enabling-metrics-ocp-operator_{context}"]
+= Enabling metrics monitoring in a {product} Operator installation on an {ocp-short} cluster
+
+You can enable and view metrics for an Operator-installed {product} instance from the *Developer* perspective of the {ocp-short} web console.
+
+.Prerequisites
+
+* Your {ocp-short} cluster has link:https://docs.openshift.com/container-platform/latest/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[monitoring for user-defined projects] enabled.
+
+* You have installed {product} on {ocp-short} using the {product} Operator.
+
+* You have installed the {ocp-short} `oc` CLI.
+
+.Procedure
+
+Currently, the {product} Operator does not support creating a `ServiceMonitor` custom resource (CR) by default. You must complete the following steps to create a `ServiceMonitor` custom resource (CR) to scrape metrics from the endpoint.
+
+. Substitute the values for your environment and then run the following commands to create the `ServiceMonitor` CR:
++
+[source,terminal]
+----
+<custom_resource_name>=my-rhdh
```

Comment:
From a user perspective, I think it is a bit unclear if they need to run this as a command or not. Instead of an example value like `my-rhdh`, could we maybe just describe the meaning of this parameter?
For example:
> Replace `<custom_resource_name>` with the name of your {product} Custom Resource.
> Replace `<project_name>` with the name of the {ocp-short} project where your {product} instance is running.

Reply:
I had actually wondered if it made sense to document the process this way :) I wasn't sure if there was a reason why you set the variables and then ran the cat command, but I think it makes more sense and is more consistent with other RH docs to document it by just telling users to create the YAML CR and then apply it :)